js/app1.js

```diff
@@ -1,121 +1,225 @@
+  }
+  return counter;
+}
 
-let favoriteFoods = ['Thai', 'Peruvian', 'Italian', 'thai', 'peruvian', 'italian'];
-// faveFoods = favoriteFoods.toLowerCase();
-let notFavoriteFoods = ['Indian', 'Chinese', 'Japanese', 'indian', 'chinese', 'japanese'];
-// notFaveFoods = notFavoriteFoods.toLowerCase();
-let numTries = 6;
-
-for (let i = 0; i < numTries; i++) {
-  let guessFoods = prompt('Which of the following cuisines are my favorites? Thai, Peruvian, Italian, Indian, Chinese, Japanese');
-  if (notFavoriteFoods.includes(guessFoods)) {
-    alert('You are incorrect! ' + guessFoods + ' is not amongst my favorite cuisines. Try again!');
-  } else if (favoriteFoods.includes(guessFoods)) {
-    alert('You are correct! I love ' + guessFoods);
-    count = count + 1;
-    break;
-  } else {
-    alert('That was not even an option!');
+function askNumber(counter){
+  let correctNum = Math.ceil(Math.random() * 10);
+  let numbTries = 4;
+
+  for (let i = 0; i < numbTries; i++) {
+
+    let guessNum = prompt('Guess a number between 1 and 10.');
+    guessNum = parseInt(guessNum);
+
+    if (guessNum < correctNum) {
+      alert('You went too low!');
+      // console.log(guessNum);
+    } else if (guessNum > correctNum) {
+      alert('You went too high!');
+      // console.log(guessNum);
+    } else if (guessNum === correctNum) {
+      alert('You are right! The right answer was ' + guessNum);
+      counter = counter + 1;
+      // console.log(guessNum);
+      return counter;
+    } else {
+      alert('Please enter a number');
+    }
   }
+
+  return counter;
+}
+
+function askMeals(counter){
+  let favoriteFoods = ['Thai', 'Peruvian', 'Italian'];
+  // faveFoods = favoriteFoods.toLowerCase();
+  let notFavoriteFoods = ['Indian', 'Chinese', 'Japanese'];
+  // notFaveFoods = notFavoriteFoods.toLowerCase();
+  let numTries = 6;
+  
+  for (let i = 0; i < numTries; i++) {
+    let guessFoods = prompt('Which of the following cuisines are my favorites? Thai, Peruvian, Italian, Indian, Chinese, Japanese');
+  
+    let inList = false;
+    // Iterate over both arrays to find a match
+    for (let j = 0; j < favoriteFoods.length; j++){
+      if (notFavoriteFoods[j].toLowerCase() === guessFoods.toLowerCase()){
+        alert('You are incorrect! ' + guessFoods + ' is not amongst my favorite cuisines. Try again!');
+        inList = true;
+      } else if (favoriteFoods[j].toLowerCase() === guessFoods.toLowerCase()) {
+        alert('You are correct! I love ' + guessFoods);
+        inList = true;
+        counter = counter + 1;
+        return (counter);
+      } 
+    }
+    if (!inList) {
+      alert('That was not even an option!');
+    }
+  }    
+  return (counter);
 }
 
+
+let count = 0;
+
+//console.log(alert);
+
+let quest1 = askName();
+askTrips();
+count = askDestinations(count);
+count = askIslands(count);
+count = askFish(count);
+count = askHead(count);
+count = askNumber(count);
+count = askMeals(count);
```

Comment:
Since you're only going to use one score keeping variable, I would actually recommend not making count an argument of all your functions. You can just call count in each of your functions and increment it there if the answer is correct. This does require your count variable to be declared before any of your functions though.